src/index.html

```diff
@@ -22,6 +22,408 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="MOYO"
+          class="logo_img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_ul">
+          <li class="nav_link">
+            <a
+              href="index.html"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_link">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav_link">
```